src/client/src/orangehrmRecruitmentPlugin/pages/EditJobVacancy.vue

```diff
@@ -475,10 +475,15 @@ export default {
         });
       })
       .then(() => {
-        this.httpAttachments.get(this.vacancyId).then(response => {
-          const {data} = response.data;
-          this.attachments = attachmentNormalizer(data);
-        });
+        this.httpAttachments
+          .request({
```

Comment:
We can't here since `id` is not passed as query param and as a part of url. But when we initate API service, we added id there we can use it then. if using request, there's no reason for `httpAttachments` to exist.